primitives/application-crypto/src/ed25519.rs

```diff
@@ -24,6 +24,7 @@ pub use primitives::ed25519::*;
 
 mod app {
 	use primitives::testing::ED25519;
+	use rstd::vec::Vec;
```

Comment:
This is not correct. The macro should internally use a full path to vec. (rstd is probably already exported by application crypto)

And then use `$crate::rstd::vec::Vec`

Reply:
Not sure I understood you right. The above actually built correctly for me (but CI complained...probably rust version?). However, I've changed it to `$crate.....$`

Reply:
see below...